### PR TITLE
feat: initial javascript api

### PR DESCRIPTION
**Pull request by donovanhiland**

### CODE REVIEW

File: example/ios/Podfile [hunk 1]
```diff
@@ -14,8 +14,8 @@ target 'MidnightExample' do
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable these next few lines.
- use_flipper!({ 'Flipper' => '0.75.1' })
- post_install do |installer|
-   flipper_post_install(installer)
- end
+  #  use_flipper!
```
Comment (donovanhiland) [inline example/ios/Podfile:17]: This was causing me issues. If we need it we can figure out what was wrong at that point.

Comment (josectobar) [reply example/ios/Podfile:17]: I don't think you need it.

**Review by josectobar — APPROVED** [review]

File: example/ios/Podfile (hunk 1 above)
Comment (manasv) [reply example/ios/Podfile:17]: For me it was solved setting the flipper version in the Podfile (or desabling it as you did haha) if we are not using it it should be safe to even delete it.

**Pull request merged**